src/App.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { useContext } from "react";
+import "./App.css";
+import { AppContext } from "./components/AppContext/AppContext";
+
+export const App = () => {
```

Comment:
Curious - is it normal convention to leave out this parent component out of the `components` dir?

Reply:
don't think it matters too much, we can move everything into components eventually